src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -20,8 +24,20 @@ fun main() {
         .build()
 
     val credentials = getObjectMapper().readValue<VaultCredentials>(Paths.get(CREDENTIALS_PATH).toFile())
+    val httpClient = ClientFactory.createHttpClient(env)
+    val accessTokenService = AccessTokenService(AccessTokenClient(env.aadAccessTokenUrl, env.clientId, credentials.clientsecret, httpClient))
```

Comment:
For å snakke med Azure trenger du en httpclient med proxy (det må settes eksplisitt på klienten, den plukker det ikke opp av seg selv :( )

Reply:
supert!